src/main/java/org/gridsuite/spreadsheetconfig/server/controller/SpreadsheetConfigController.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.spreadsheetconfig.server.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.gridsuite.spreadsheetconfig.server.SpreadsheetConfigApi;
+import org.gridsuite.spreadsheetconfig.server.dto.SpreadsheetConfigDto;
+import org.gridsuite.spreadsheetconfig.server.service.SpreadsheetConfigService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+/**
+ * @author Achour BERRAHMA <achour.berrahma at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + SpreadsheetConfigApi.API_VERSION + "/spreadsheet-configs")
+@RequiredArgsConstructor
+@Tag(name = "Spreadsheet Config", description = "Spreadsheet Configuration API")
+public class SpreadsheetConfigController {
+
+    private final SpreadsheetConfigService spreadsheetConfigService;
+
+    public static final String DUPLICATE_FROM = "duplicateFrom";
+
+    @PostMapping(consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a new spreadsheet configuration",
+            description = "Creates a new spreadsheet configuration and returns its ID")
+    @ApiResponse(responseCode = "201", description = "Configuration created",
```

Comment:
Why do we start using more specific return code ? We used to stick to 200 for OK (even for post). For the sake of harmonization shouldn't we keep the same logic everywhere (or change it everywhere but not here in this PR) ?

Reply:
We can keep it like this here, it's a good practice to return the right code :)